README.md

```diff
@@ -1,36 +1,99 @@
+>
+> However, we do **not** recommend using Pre-Preview software in production apps. Software in this status will change as we add features and iterate based on feedback.
 
-# API documentation
+We are still working on adding several features to bring this SDK to parity with the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (`xmtp-js`).
 
-- [Documentation for the main branch](https://github.com/expo/expo/blob/main/docs/pages/versions/unversioned/sdk/xmtp-react-native-sdk.md)
-- [Documentation for the latest stable release](https://docs.expo.dev/versions/latest/sdk/xmtp-react-native-sdk/)
+Currently, these are the features supported by this SDK:
 
-# Installation in managed Expo projects
+- Create a `Client` from an ethers `Signer`
+- List conversations
+- List messages (unpaginated for now) in a conversation
+- Start a conversation
 
-For [managed](https://docs.expo.dev/versions/latest/introduction/managed-vs-bare/) Expo projects, please follow the installation instructions in the [API documentation for the latest stable release](#api-documentation). If you follow the link and there is no documentation available then this library is not yet usable within managed projects &mdash; it is likely to be included in an upcoming Expo SDK release.
+Here's a [list of features and issues](https://github.com/xmtp/xmtp-react-native/issues) we're still working on. 
 
-# Installation in bare React Native projects
+Is there a feature you need that's not on the list? Please [open an issue](https://github.com/xmtp/xmtp-react-native/issues).
 
-For bare React Native projects, you must ensure that you have [installed and configured the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+Or better yet, open a PR and we'll get it reviewed and merged as soon as possible. If you contribute a PR that gets merged into this repo, you'll be eligible to [claim this XMTP contributor POAP](https://www.gitpoap.io/gp/1042)!
 
-### Add the package to your npm dependencies
+To learn more about XMTP and get answers to frequently asked questions, see [FAQ about XMTP](https://xmtp.org/docs/dev-concepts/faq).
+
+![x-red-sm](https://user-images.githubusercontent.com/510695/163488403-1fb37e86-c673-4b48-954e-8460ae4d4b05.png)
+
+## Example apps
+
+This repo provides the following example apps that you can use to experiment and explore implementation details:
 
+- [Android example](example/android)
+
+- [iOS example](example/ios)
+
+To use an example app, run:
+
+```bash
+cd example
+npm install --force
+npm run [ios or android]
 ```
+
+## Install in a managed Expo project
+
+<!--eng: When I run this at the root of the project, I get this prompt: The android and ios projects are malformed, would you like to clear the project files and reinitialize them? › (Y/n) I imagine this is because I'm not running it in a managed Expo project. Just wanted to run it to see what happened. =)-->
```

Comment:
Ya this is a command you run from inside an actual app, not the library we've got here.